packages/app/components/document/form-actions.tsx

```diff
@@ -79,7 +83,12 @@ const FormActions = ({
         }
     }
 
-    function handleSave() {
+    async function handleSave(largeModel: boolean) {
+        if (largeModel) {
+            // This hack is in place to handle RJSF's ajv8 validator performance issues.
+            await wait(500)
```

Comment:
@benforshey save performance isn't as noticeable after upgrading AJV, reducing this save delay